src/Uno.UI/UI/Xaml/FrameworkElement.Interface.wasm.cs

```diff
@@ -204,29 +203,20 @@ private void RecalculateBrushOnSizeChanged(bool shouldRecalculate)
 
 		public event DependencyPropertyChangedEventHandler IsEnabledChanged;
 
+		[GeneratedDependencyProperty(DefaultValue = true, ChangedCallback = true, CoerceCallback = true, Options = FrameworkPropertyMetadataOptions.Inherits)]
+		public static readonly DependencyProperty IsEnabledProperty = CreateIsEnabledProperty();
+
 		public bool IsEnabled
 		{
-			get { return (bool)GetValue(IsEnabledProperty); }
-			set { SetValue(IsEnabledProperty, value); }
+			get => GetIsEnabledValue();
+			set => SetIsEnabledValue(value);
 		}
 
-		public static readonly DependencyProperty IsEnabledProperty =
-			DependencyProperty.Register(
-				name: "IsEnabled",
-				propertyType: typeof(bool),
-				ownerType: typeof(FrameworkElement),
-				typeMetadata: new FrameworkPropertyMetadata(
-					defaultValue: true,
-					options: FrameworkPropertyMetadataOptions.Inherits,
-					propertyChangedCallback: (s, e) =>
-					{
-						var elt = (FrameworkElement)s;
-						elt?.OnIsEnabledChanged((bool)e.OldValue, (bool)e.NewValue);
-						elt?.IsEnabledChanged?.Invoke(s, e);
-					},
-					coerceValueCallback: (s, v) => (s as FrameworkElement)?.CoerceIsEnabled(v)
-				)
-		);
+		protected virtual void OnIsEnabledChanged(DependencyPropertyChangedEventArgs args)
+		{
+			OnIsEnabledChanged((bool)args.OldValue, (bool)args.NewValue);
```

Comment:
Codacy found an issue: [Refactor this method to add validation of parameter 'args' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5714227)